stylesheets/partials/units/_button.scss

```diff
@@ -10,6 +10,11 @@ button {
 
 .btn-primary {
 	background-color: $teal;
+	border: 1px solid $gray;
+	box-shadow: 0px 1px 5px rgba(0, 0, 0, 0.5);
```

Comment:
There's a _shadows.scss file with some shadow vars. You should use one of those, or add yours if it doesn't exist.

Reply:
done